packages/daemon/src/runner/job-runner.ts

```diff
@@ -244,7 +244,11 @@ export class JobRunner {
     const modelPath = path.join(this.opts.runnable.workingDir, 'model');
 
     await this.runDataCollect(dataDir, modelPath);
-    const dataset = await this.runDataAccess(dataDir);
+
+    // move plugin data directory data to job's data directory
+    await fs.copy(dataDir, this.opts.runnable.dataDir);
```

Comment:
If use fs.copyFile with flag `fs.constants.COPYFILE_FICLONE`, this operation will be more faster.

Reply:
done